src/acceleration/config/AccelerationConfiguration.hpp

```diff
@@ -117,6 +117,34 @@ class AccelerationConfiguration : public xml::XMLTag::Listener {
     std::string           preconditionerType;
   } _config;
 
+  const struct DefaultValuesIQN {
+    double      relaxationFactor           = 0.1;
+    int         maxIterationsUsed          = 100;
+    int         timeWindowsReused          = 10;
+    int         filter                     = Acceleration::QR2FILTER;
+    double      singularityLimit           = 1e-2;
+    std::string preconditionerType         = "residual-sum";
+    int         precond_nbNonConstTWindows = -1;
+  } _defaultValuesIQNILS;
+
+  const struct DefaultValuesIMVJ {
+    double      relaxationFactor           = 0.1;
+    int         maxIterationsUsed          = 10;
```

Comment:
```suggestion
    int         maxIterationsUsed          = 20;
```
Isn't 10 a bit too small?